server/src/main/java/com/hedera/block/server/BlockStreamService.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Hedera Block Node
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server;
+
+import com.google.protobuf.Descriptors;
+import com.hedera.block.protos.BlockStreamServiceGrpcProto;
+import com.hedera.block.server.producer.ProducerBlockStreamObserver;
+import com.hedera.block.server.mediator.StreamMediator;
+import com.hedera.block.server.consumer.LiveStreamObserverImpl;
+import com.hedera.block.server.consumer.LiveStreamObserver;
+import io.grpc.stub.StreamObserver;
+import io.helidon.webserver.grpc.GrpcService;
+
+import java.util.logging.Logger;
```

Comment:
I don't think we've made a decision here: java.System.Logger, java.util.logging.Logger, or SLF4J?

Reply:
I think JUL is fine. I'm using java.System.Logger in the Helidon plugin (I think Helidon is using it which is why I went down that path). I think JUL is fine, or java.System.Logger is fine, and both interoperate.

Reply:
https://stackoverflow.com/questions/59976828/difference-between-java-util-logging-logger-and-java-lang-system-logger

Reply:
Given this, I think java.lang.System.Logger is probably better.

Reply:
fixed